process-phoenix/src/main/java/com/jakewharton/processphoenix/ProcessPhoenix.java

```diff
@@ -53,10 +53,16 @@ public static void triggerRebirth(Context context) {
    * <p>
    * Behavior of the current process after invoking this method is undefined.
    */
-  public static void triggerRebirth(Context context, Intent... nextIntents) {
+  public static void triggerRebirth(Context context, Intent... intents) {
+    if (intents.length < 1) {
```

Comment:
By adding this line you are breaking backwards compatibility. As per the Readme, `ProcessPhoenix.triggerRebirth(context);` is a perfectly valid way of triggering the "rebirth".